tutorials/R/download-ais-data/download-NEON-AIS-data.Rmd

```diff
@@ -0,0 +1,307 @@
+* `token`: this allows you to input your NEON API token to obtain faster downloads. 
+Learn more about NEON API tokens in the <a href="https//:www.neonscience.org/neon-api-tokens-tutorial" target="_blank">**Using an API Token when Accessing NEON Data with neonUtilities** tutorial</a>. 
+
+There are additional inputs you can learn about in the 
+<a href="https//:www.neonscience.org/neonDataStackR" target="_blank">**Use the neonUtilities R Package to Access NEON Data** tutorial</a>. 
+
+The `dpID` is the data product identifier of the data you want to 
+download. The DPID can be found on the 
+<a href="http://data.neonscience.org/data-products/explore" target="_blank">
+Explore Data Products page</a>.
+
+It will be in the form DP#.#####.###. For this tutorial, we'll use three data
+product: 
+
+* DP1.20288.001: Water quality
+
+We will want data from only one NEON field site, Blacktail Deer Creek, WY (BLDE)
+from October and November 2019. 
+
+Now we can download our data. If you are not using a NEON token to download your
+data, remove the `token = Sys.getenv(NEON_TOKEN)` line of code. We have 
+`check.size = F` so that the script runs well but remember you always want to 
+check your download size first. This code downloads 3.6 MiB and 289.4 KiB 
+respectively.
+
+```{r download-data}
+# download data of interest - Water Quality
+waq<-loadByProduct(dpID="DP1.20288.001", site="BLDE", 
+										 startdate="2019-10", enddate="2019-11", 
+										 package="expanded", 
+										 token = Sys.getenv("NEON_TOKEN"),
+										 check.size = F)
+
+```
+
+<div id="ds-challenge" markdown="1">
+### Challenge: Download Nitrate Data
+  
+Using what you've learned above, can you modify the code to download data for the following parameters. 
+
+* Data Product DP1.20033.001: nitrate in surface water
+* The expanded data tables
+* Dates matching the other data products you've downloaded
+
+1. What is the size of the downloaded data? 
+2. Without downloading all the data, how can you tell what the difference in size
+between the "expanded" and "basic" packages is? 
+</div>
+
+```{r challenge-code-extent-crs, echo=FALSE}
+# download data of interest - Nitrate in Suface Water
+nitr<-loadByProduct(dpID="DP1.20033.001", site="BLDE", 
+											 startdate="2019-10", enddate="2019-11", 
+											 package="basic", 
+											 token = Sys.getenv("NEON_TOKEN"),
+											 check.size = F)
+
+# #1. 73.3 KiB 
+# #2. You can change check.size to True (T), and compare "basic" vs "expaneded"
+# package types. The basic package is 66.2 KiB. 
+
+```
+
+## Files Associated with Downloads
+
+The data we've downloaded comes as an object that is a named list of objects. 
+To work with each of them, select them from the list using the `$` operator. 
+
+```{r loadBy-list}
+# view all components of the list
+names(waq)
+
+# View the dataFrame
+View(waq$waq_instantaneous)
+
+```
+
+We can see that there are four objects in the downloaded nitrate data. One 
```

Comment:
From Bobby: I think this should be downloaded "water quality" data.  [MJ: yep, I changed the product midway through development and didn't replace correctly]

Reply:
Done